Users/views.py

```diff
@@ -42,18 +42,33 @@ def post(self, request, *args, **kwargs):
         return Response(serializer.validated_data, status=status.HTTP_200_OK)
 
 
+from rest_framework_simplejwt.tokens import AccessToken
+
 class UserLoginView(APIView):

```

Comment:
If you create the user login view to obtain the token, what is the use of the custom token obtain view?